nixos/modules/services/networking/mtr-exporter.nix

```diff
@@ -0,0 +1,77 @@
+{ config, lib, pkgs, ... }:
+
+let
+  inherit (lib)
+    maintainers types mkEnableOption mkOption mkIf
+    literalExpression escapeShellArg escapeShellArgs;
+  cfg = config.services.mtr-exporter;
+in {
+  options = {
+    services = {
+      mtr-exporter = {
+        enable = mkEnableOption "a Prometheus exporter for MTR.";
```

Comment:
a trailing `.` is already added by `mkEnableOption`, adding another ends up as `[...] for MTR..`

Reply:
And I assume `mkOption` doesn't? That's kinda inconsistent.

Reply:
indeed it doesn't. it does make sense here though to include the full head and tail for a sentence and only insert parts that won't usually be there anyway, so wouldn't say the inconsistency is all that important.